### DISCUSSION TITLE
3D Love!

### DISCUSSION THREAD

**Discussion by RadTechDad** [keiwando/toonsquid#27] [Ideas]
I'd love to be able to import 3D models (fbx, glTF, obj) and have them displayed as 2d sprites.

Similar to Moho Animation Software:

![Moho Animation Software](https://cdn.shopify.com/s/files/1/0552/8741/4966/files/kukulkan2.gif)

**Comment by trinhpxl** [comment]
can the ipad even do this 😳

**Reply by RadTechDad** [reply]
Probably the newer ones.

**Comment by keiwando** [comment]
While I currently don't have plans for full 3D editing support in ToonSquid, adding the ability to import 3D animations into something like a dedicated 3D layer would be an interesting first step to try in the future.
I'm definitely curious to see how many users would like to see something like this, especially if you wouldn't actually be able to edit any of the 3D animation yet.

**Comment by Quentie** [comment]
Actually, exporting as glTF would be awesome—maybe even better than a Lottie export, since it would allow us to bring projects into Blender or any 3D engine of our choice. Similar to how Moho 14 handles it: [Moho 14 glTF export tutorial](https://tutorial.lostmarble.com/app/page/1f3KuR0FR0jsmLYR2H_MQrAsgR7sxknNy7RHy9ctzzlg?p=1Zvlrp5i9FSFV3NLXVt88rSO0OGTFEhsr)
